e2e/specs/menu_bar/file_menu.test.js

```diff
@@ -39,4 +39,28 @@ describe('file_menu/dropdown', function desc() {
             settingsWindow.should.not.be.null;
         }
     });
+
+    it('MM-T804 Preferences in Menu Bar open the Settings page', async () => {
+        const mainWindow = this.app.windows().find((window) => window.url().includes('index'));
+        mainWindow.should.not.be.null;
+        if (process.platform === 'win32' || process.platform === 'linux') {
```

Comment:
Can we move this check to the outside of the test and just check that `process.platform !== 'darwin'`?

Reply:
✅ Done as requested! Thanks